### DISCUSSION TITLE
Documentation up on the supabase website

### DISCUSSION THREAD

**Discussion by jan-tennert** [supabase-community/supabase-kt#141] [Announcements]
As the title says, the new [documentation](https://supabase.com/docs/reference/kotlin/introduction) is up on the supabase website. 🎉

![image](https://github.com/supabase-community/supabase-kt/assets/26686035/9cbb6e5f-a98e-4804-8d4c-5094a8188717)

**Comment by TerryQuiet** [comment]
the link is missing here
https://supabase.com/docs/guides/api/rest/client-libs

**Reply by jan-tennert** [reply]
Will fix that later 👍
